src/main/java/org/springframework/data/repository/core/support/RepositoryInvocationMulticaster.java

```diff
@@ -56,7 +56,7 @@ public void notifyListeners(Method method, Object[] args, RepositoryMethodInvoca
 
 	/**
 	 * {@link RepositoryInvocationMulticaster} implementation that notifies {@link RepositoryMethodInvocationListener}
-	 * upon {@link #notifyListeners(Method, Object[], RepositoryMethodInvocationResult)}.
```

Comment:
I just fixed this broken JavaDoc which I noticed along the way, it's not directly related to this pr.